must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -156,3 +156,16 @@ echo "collecting dump of oc get volumereplication all namespaces" | tee -a  "${B
 echo "collecting dump of oc get volumereplicationgroups all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
 { oc get volumereplicationgroups --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/vrg_all_namespaces"
 { oc adm --dest-dir="${BASE_COLLECTION_PATH}/namespaces/all/" inspect vrg --all-namespaces --"${SINCE_TIME}"; } >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
+
+# Collect details of storageclassclaim for managed services
+ echo "collecting dump of oc get storageclassclaim" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
+namespaces=$(oc get storageclassclaim -A --no-headers | awk '{print $1}')
```

Comment:
why not `oc get namespaces` here?

Reply:
`oc get namespaces` shall list all the namespaces right? How do we get under which ns storageclassclaim is created? Am I missing anything?

Reply:
got it. it should be fine. this avoids listing all namespaces which doesn't have storageclassclaim

Reply:
yes.